doc/sphinx/language/core/inductive.rst

```diff
@@ -8,14 +8,15 @@ Inductive types
 
 .. cmd:: Inductive @inductive_definition {* with @inductive_definition }
 
-   .. insertprodn inductive_definition cumul_ident_decl
+   .. insertprodn inductive_definition cumul_univ_decl
 
    .. prodn::
       inductive_definition ::= {? > } @cumul_ident_decl {* @binder } {? %| {* @binder } } {? : @type } {? := {? @constructors_or_record } } {? @decl_notations }
       constructors_or_record ::= {? %| } {+| @constructor }
       | {? @ident } %{ {*; @record_field } {? ; } %}
       constructor ::= @ident {* @binder } {? @of_type }
       cumul_ident_decl ::= @ident {? @cumul_univ_decl }
+      cumul_univ_decl ::= @%{ {* {? {| = | + | * } } @ident } {? + } {| %| {*, @univ_constraint } {? + } %} | {| %} | %|%} } }
```

Comment:
I think it's best to leave `cumul_univ_decl` where it has been put in #12653, i.e. next to `univ_decl`.

Reply:
As you wish.  That list of productions in the universe chapter is a bit of an anomaly, throwing together several nonterminals that don't all refer to each other.

Reply:
We should improve this, of course, but there is no point in putting productions very far from the place they are currently documented.